src/misc/lv_fs.c

```diff
@@ -493,15 +499,27 @@ const char * lv_fs_get_last(const char * path)
  *   STATIC FUNCTIONS
  **********************/
 
+/**
+ * Checks if the path starts with a drive prefix (i.e. "X:...")
+ * @param path path string (E.g. S:/folder/file.txt)
+ * @return true if path has drive letter
+ */
+static bool lv_fs_path_has_drive(const char * path)
+{
+    if(path[0] == '\0') return false;
+
+    return ('A' <= path[0] && path[0] <= 'Z') && path[1] == ':';
+}
+
 /**
  * Skip the driver letter and the possible : after the letter
  * @param path path string (E.g. S:/folder/file.txt)
  * @return pointer to the beginning of the real path (E.g. /folder/file.txt)
  */
 static const char * lv_fs_get_real_path(const char * path)
 {
-    path++; /*Ignore the driver letter*/
-    if(*path == ':') path++;
+    if(lv_fs_path_has_drive(path))
+        path += 2;
```

Comment:
Looks like it will break file paths missing the `:`, like `A/path/to/file`. I think this is ok.